### ISSUE TITLE
FeatureComplete: implement links predicate

### ISSUE THREAD

**Issue by mcharsley** [uutils/findutils#10]

**Comment by hanbings** [comment]
Hi! Are you still interested in this issues?
It looks like it already has some level of implementation:

```shell
$ ./target/debug/find . -links 1
$ ./target/debug/find . -links +0
$ ./target/debug/find . -links -2
```

**Comment by tavianator** [comment]
This was done by https://github.com/uutils/findutils/pull/167, I just forgot to add the "closes #10" tag :)

**Comment by hanbings** [comment]
Thanks. Now let's close it!
